components/Filter/FilterBy/Collection.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Heading,
+  Icon,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  SkeletonCircle,
+  SkeletonText,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react'
+import { HiOutlineSearch } from '@react-icons/all-files/hi/HiOutlineSearch'
+import { convertCollection } from 'convert'
+import useTranslation from 'next-translate/useTranslation'
+import { FC, useMemo, useState } from 'react'
+import { UseFormReturn } from 'react-hook-form'
+import {
+  CollectionFilter,
+  StringFilter,
+  useSearchCollectionQuery,
+} from '../../../graphql'
+import { Filter } from '../../../hooks/useAssetFilterFromQuery'
+import CollectionListItem from '../../Collection/ListItem'
+import List, { ListItem } from '../../List/List'
+
+type Props = {
+  formValues: UseFormReturn<Filter, any, undefined>
+  selectedCollection?: { chainId: number; address: string }
+  onFilterChange: (data?: Partial<Filter>) => void
+  setCollection: (data?: { chainId: number; address: string }) => void
```

Comment:
Need to rename this function now that the component is split:
```suggestion
  onCollectionChange: (data?: { chainId: number; address: string }) => void
```

Reply:
[Change naming onCollectionChange](https://github.com/liteflow-labs/starter-kit/pull/375/commits/90e068b0f44dfb0935522771ee4a8cd153cd3649)
[Fix missing prop change](https://github.com/liteflow-labs/starter-kit/pull/375/commits/27fa14642cc98130d37176dd95dd8125a6491546)